sjb/config/test_cases/push_jenkins_images.yml

```diff
@@ -1,63 +1,114 @@
 ---
-parent: 'test_cases/test_branch_jenkins_images.yml'
```

Comment:
doesn't this mean test_branch_jenkins is still going to be broken, which means PR tests will still fail?

Reply:
yep I need to move the s2i bits (minus building of s2i) into test_branch_jenkins

will reswizzle in a bit

Reply:
well ... just realized a few things ... depends on which release branch I think
- 3.11 and 4.0 use ci-operator and not aos-cd-jobs
- we are generally only updating those two for security advisories ...
- ... but for say plugin/image fixes outside of that, we'd use test_branch_jenkins ..
- but we'd also use the unit tests and s2 there ... 

So probably just need to keep the s2i build in play, and update the golang

agree @bparees ?